readme.txt

```diff
@@ -125,6 +125,7 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Removed the `type="submit"` from the button element in the ticket quantity remove template. [ETP-224]
 * Fix - Removed the duplicate `type="button"` from the button element in the ticket quantity add template. [ETP-224]
 * Fix - Correct the `Total` value, when the WooCommerce options for currency are empty (Thousand and decimal separator, number of decimals). [ETP-231]
+* Fix - Load plugin text domain on the new `tribe_load_text_domain` action hook, which fires on 'init' instead of on the 'plugins_loaded' hook. [ET-739]
```

Comment:
```suggestion
* Fix - Load plugin text domain on the new `tribe_load_text_domains` action hook, which fires on 'init' instead of on the 'plugins_loaded' hook. [ET-739]
```